src/java.base/share/classes/java/time/format/DateTimeFormatter.java

```diff
@@ -502,7 +503,10 @@
  * {@code LocalDateTime} to form the instant, with any zone ignored.
  * If a {@code ZoneId} was parsed without an offset then the zone will be
  * combined with the {@code LocalDateTime} to form the instant using the rules
- * of {@link ChronoLocalDateTime#atZone(ZoneId)}.
+ * of {@link ChronoLocalDateTime#atZone(ZoneId)}. If the {@code ZoneId} was
+ * parsed from the zone name that does not imply daylight saving time, then
+ * {@link ChronoZonedDateTime#withLaterOffsetAtOverlap()} is issued
+ * to use the standard offset at the overlap, before forming the instant.
```

Comment:
Is the standard offset the subject instead of the withLaterOffsetAtOverlap method? Calling the method seems to be an impl detail to me. We might rephrase the sentence to sth. like:
If the {@code ZoneId} parsed does not indicate daylight saving time, the standard offset will be used at the local time-line overlap as specified in the {@link ChronoZonedDateTime#withLaterOffsetAtOverlap()} method to form the instant.

Reply:
Removed the method, and made `standard offset` the subject of the sentence.